nearswap/tests/fee_simulation.rs

```diff
@@ -0,0 +1,282 @@
+    );
+
+    // Pool creation by root account
+    call!(
+        root,
+        nearswap.create_pool(to_va("dai".into())),
+        deposit = to_yocto("1")
+    )
+    .assert_success();
+
+    // Register account lp1
+    call!(
+        lp1,
+        nearswap.storage_deposit(None, Some(true)),
+        deposit = to_yocto("1")
+    )
+    .assert_success();
+    // Register account lp2
+    call!(
+        lp2,
+        nearswap.storage_deposit(None, Some(true)),
+        deposit = to_yocto("1")
+    )
+    .assert_success();
+    print!("Account Registered!");
+    // Deposit near in account deposit lp1
+    call!(
+        lp1,
+        nearswap.storage_deposit(None, None),
+        deposit = to_yocto("35")
+    )
+    .assert_success();
+    // Deposit near in account deposit lp2
+    call!(
+        lp2,
+        nearswap.storage_deposit(None, None),
+        deposit = to_yocto("35")
+    )
+    .assert_success();
+
+    // Add to accounts whitelist
+    call!(
+        lp1,
+        nearswap.add_to_account_whitelist(&vec![to_va(dai())])
+    )
+    .assert_success();
+    call!(
+        lp2,
+        nearswap.add_to_account_whitelist(&vec![to_va(dai())])
+    )
+    .assert_success();
+
+    // Depositing tokens in account deposit
+    call!(
+        lp1,
+        token1.ft_transfer_call(to_va(clp_contract()), to_yocto("100").into(), None, "".to_string()),
+        deposit = 1
+    )
+    .assert_success();
+    call!(
+        lp2,
+        token1.ft_transfer_call(to_va(clp_contract()), to_yocto("100").into(), None, "".to_string()),
+        deposit = 1
+    )
+    .assert_success();
+
+    print!("Token deposited in contract account deposit");
+
+    // Adding liquidity
+    let lp1_shares = call!(
+        lp1,
+        nearswap.add_liquidity(dai(), to_yocto_str("9"), to_yocto_str("90"), U128(0)),
+        deposit = 1
+    )
+    .unwrap_json::<U128>();
+
+    let lp2_shares = call!(
+        lp2,
+        nearswap.add_liquidity(dai(), to_yocto_str("1"), to_yocto_str("10.01"), U128(0)),
+        deposit = 1
+    )
+    .unwrap_json::<U128>();
+
+    print!("Added Liquidity!");
+    // Register alice before swapping
+    call!(
+        alice,
+        nearswap.storage_deposit(None, Some(true)),
+        deposit = to_yocto("1")
+    )
+    .assert_success();
+    // Deposit near in account deposit
+    call!(
+        alice,
+        nearswap.storage_deposit(None, None),
+        deposit = to_yocto("5")
+    )
+    .assert_success();
+    call!(
+        alice,
+        nearswap.add_to_account_whitelist(&vec![to_va(dai())])
+    )
+    .assert_success();
+    print!("Alice Registered!");
+    // Alice buy token by paying 5 NEAR
+    let mut before_swap_token = view!(
+        nearswap.get_deposit_token(alice.account_id.clone(), dai())
+    ).unwrap_json::<U128>();
+    assert_close(before_swap_token, to_yocto("0"), 0);
+
+    let price_n2t = view!(
+            nearswap.price_near_to_token_in(dai(), to_yocto_str("5"))
```

Comment:
I'm not sure if `price_` prefix has sense any more.

Reply:
`near_to_token_in` doesn't seem very clear though it indicates it could be swap or price